docs/sql-migration-guide-upgrade.md

```diff
@@ -10,6 +10,9 @@ displayTitle: Spark SQL Upgrading Guide
 ## Upgrading From Spark SQL 2.4 to 3.0
 
   - In PySpark, when creating a `SparkSession` with `SparkSession.builder.getOrCreate()`, if there is an existing `SparkContext`, the builder was trying to update the `SparkConf` of the existing `SparkContext` with configurations specified to the builder, but the `SparkContext` is shared by all `SparkSession`s, so we should not update them. Since 3.0, the builder comes to not update the configurations. This is the same behavior as Java/Scala API in 2.3 and above. If you want to update them, you need to update them prior to creating a `SparkSession`.
+  - In Avro data source, the function `from_avro` supports following parse modes:
```

Comment:
nit: let's add a space to make it more readable (I added each space between each item).